<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1514071040065' stop='1514071047819'>
    <name>Testing the report and log</name>
    <title>Testing the report and log</title>
    <test-cases>
        <test-case start='1514071040065' status='passed' stop='1514071044929'>
            <name>validating correct title</name>
            <title>validating correct title</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='fa46f945-9990-4618-a850-432613ae8418-attachment.png' type='image/png' size='38085'/>
            </attachments>
        </test-case>
        <test-case start='1514071044929' status='failed' stop='1514071047818'>
            <name>validating in-correct title</name>
            <title>validating in-correct title</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='cfe3e123-0faa-4561-ab49-88e649e5a9a1-attachment.png' type='image/png' size='38085'/>
            </attachments>
            <failure>
                <message>Expected 'Protractor practice website - Banking App' to contain 'ProtractorOne'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (/Users/trupti/WebstormProjects/Protractor_Test/validate_spec.js:20:23)
                    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
                    at new ManagedPromise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
                    at ControlFlow.promise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
                    at schedulerExecute (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2974:25)
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>